templates/email/default/alert-update.html

```diff
@@ -14,6 +14,7 @@
   [% start_padded_box | safe %]
   <h1 style="[% h1_style %]">New updates on <a href="[% problem_url %]">[% title %]</a></h1>
   [%~ INCLUDE '_email_comment_list.html' %]
+  [%~ INCLUDE '_alert_update_after_updates.html' %]
```

Comment:
This needs a TRY/CATCH/END wrapper around it like the other INCLUDE below in the same file, otherwise it will error in other cobrands where the template does not exist.